kotlin/src/main/java/app/rive/runtime/kotlin/RiveAnimationView.kt

```diff
@@ -30,15 +30,16 @@ import java.util.*
  */
 class RiveAnimationView(context: Context, attrs: AttributeSet? = null) : View(context, attrs),
     Observable<RiveDrawable.Listener> {
-    // TODO: stop creating an empty drawable to begin with.
-    private var _drawable: RiveDrawable = RiveDrawable();
-    private var resourceId: Int? = null;
-
-    var drawable: RiveDrawable
-        get() = _drawable
-        private set(value) {
-            _drawable = value
-        }
+    // There's always just one drawable associated with an animation view
+    val drawable = RiveDrawable()
```

Comment:
So here I've tried to re-use the drawable that's created when the View is instantiated